### PR TITLE
Prevent copying of apks in `built/`

### PR DESCRIPTION
**Pull request by kenany**
The `webdriver-js-extender@2.1.0` tarball contains an excessively deep
series of `built/` folders:

```bash
$ npm pack webdriver-js-extender
npm http fetch GET 200 https://registry.npmjs.org/webdriver-js-extender/-/webdriver-js-extender-2.1.0.tgz 85ms
npm notice 📦  webdriver-js-extender@2.1.0
npm notice === Tarball Contents ===
npm notice 1.0kB package.json
npm notice 250B  .travis.yml
npm notice 649B  CONTRIBUTING.md
npm notice 1.5kB gulpfile.js
npm notice 1.1kB LICENSE
npm notice 582B  README.md
npm notice 289B  tsconfig.json
npm notice 0     built/built/built/built/built/built/built/built/built/built/built/built/built/built/built/built/spec/command_tests/totally_real_apk.apk
npm notice 0     built/built/built/built/built/built/built/built/built/built/built/built/built/built/built/spec/command_tests/totally_real_apk.apk
npm notice 0     built/built/built/built/built/built/built/built/built/built/built/built/built/built/spec/command_tests/totally_real_apk.apk
npm notice 0     built/built/built/built/built/built/built/built/built/built/built/built/built/spec/command_tests/totally_real_apk.apk
npm notice 0     built/built/built/built/built/built/built/built/built/built/built/built/spec/command_tests/totally_real_apk.apk
npm notice 0     built/built/built/built/built/built/built/built/built/built/built/spec/command_tests/totally_real_apk.apk
npm notice 0     built/built/built/built/built/built/built/built/built/built/spec/command_tests/totally_real_apk.apk
npm notice 0     built/built/built/built/built/built/built/built/built/spec/command_tests/totally_real_apk.apk
npm notice 0     built/built/built/built/built/built/built/built/spec/command_tests/totally_real_apk.apk
npm notice 0     built/built/built/built/built/built/built/spec/command_tests/totally_real_apk.apk
npm notice 0     built/built/built/built/built/built/spec/command_tests/totally_real_apk.apk
npm notice 0     built/built/built/built/built/spec/command_tests/totally_real_apk.apk
npm notice 0     built/built/built/built/spec/command_tests/totally_real_apk.apk
npm notice 0     built/built/built/spec/command_tests/totally_real_apk.apk
npm notice 0     built/built/spec/command_tests/totally_real_apk.apk
npm notice 533B  built/lib/command_definition.d.ts
```

This is because the `copy` task will create a new level every time it
copies an apk that is already in `built/`. This patch fixes this issue
by ensuring that apks in `built/` are never copied.

### CODE REVIEW

**Comment by Trojaner** [comment]
Any particular reason this wasn't merged yet?

**Comment by brunhil** [comment]
Any input @NickTomlin ? Getting this fix merged in would greatly help us out.

**Comment by brunhil** [comment]
@qiyigg - Any help?

**Comment by qiyigg** [comment]
add @vikerman, who will take over the Protractor stuff.

**Comment by brunhil** [comment]
> When can we expect this to be merged? Because of that Windows is failing copying profile on my remote machine and support points to those folders as a reason.
> webdriver-js-extender problematic folder structure

@Marooned-MB , we found a workaround where we were able to call robocopy to clear out the nested folders against an empty folder. This should get you unstuck at least for the time being. 

`robocopy C:\Temp $PathToNodeModules /purge`

**Comment by dichovsky** [comment]
When can we expect this to be merged?

**Comment by quantranld** [comment]
Encounter this issue on my machine today. Pls merge the fix.

**Comment by Rubicen** [comment]
Is this going to get merged? This is still a problem and causing me strife with builds.

**Review by bb1 — APPROVED** [review]
Perfect.

**Comment by oehm-smith** [comment]
So close.  Can someone please hit the button!

**Comment by Rubicen** [comment]
Bumping this again in hope

**Comment by Harsha509** [comment]
HI @NickTomlin , can you please review the PR and merge if all checks looks good. :)

**Comment by mkArtakMSFT** [comment]
Still waiting on this. Is there a good reason for not accepting this PR?

**Comment by JosephBittman** [comment]
Please merge this. This is such a hassle to deal with.

**Comment by smande00** [comment]
Adding another vote for merging.

**Comment by mafster** [comment]
XD please tell me this is some linux troll just having a good old laugh at windows users battling their filepath limit issues?

p.s. current job using windows :( i need the fix too.

**Comment by nnit-TroelsJensen** [comment]
Merge, please?

**Comment by tudorsibiu90** [comment]
This is still an issue for me. How do I proceed?

**Comment by Denyos** [comment]
I'm encountering the same issue. Please merge this fix so I'm not force to trick all paths.

**Comment by Ragha2207** [comment]
Please merge this fix

**Comment by plinkpaste** [comment]
@bb1 What's going on with this?

**Comment by TobiasMarklund** [comment]
No love for this PR since it was approved 1 year ago? Can someone please check this, push the merge button and do a release? It causes massive headaches on windows when creating angular applications.
Ping @kyliau and @heathkit since they seem to be committing to Protractor these days. The people mentioned above have no activity there.

**Comment by israelvaldez** [comment]
I added this to my windows build agents provisioning scripts and haven't had this problem in a while, however the problem being fixed would be nice.
`$path = 'HKLM:\SYSTEM\CurrentControlSet\Control\FileSystem'
$key = try {
    Get-Item -Path $path -ErrorAction Stop
}
catch {
    New-Item -Path $path -Force
}
try{
  New-ItemProperty -Path $key.PSPath -Name LongPathsEnabled -Value 1 -ErrorAction Stop
}
catch{
  Set-ItemProperty -Path $key.PSPath -Name LongPathsEnabled -Value 1
}`

**Review by kyliau — APPROVED** [review]

**Comment by kyliau** [comment]
Change LGTM. I'll have to ask the team for admin access to this repo then I'll make a release this week.

**Comment by Marooned-MB** [comment]
Whoa! After almost 2 years we have this PR approved. What a day! :)

**Comment by kyliau** [comment]
Thank you @TobiasMarklund for alerting me on this issue, and sorry for the long wait!

**Pull request merged**
